src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -63,17 +91,30 @@ def __init__(
         @param password: the password of the nso account
```

Comment:
You should probably update the additional parameters in the docstring here.

Reply:
different PR